### PR TITLE
project : Rock paper scissors

### PR DESCRIPTION
**Pull request by nshanbhag20001412**

### CODE REVIEW

**Comment by Harshil-Jani** [comment]
@nshanbhag20001412 You need to create another branch  and do your commits there, not in the master or main branch. Other things are good to go ! :+1:

**Review by Harshil-Jani — COMMENTED** [review]
Code is ready to be merged ! Just uncommit from main branch and commit in new branch.

**Review by soumik2012 — APPROVED** [review]
LGTM 💯 @nshanbhag20001412

**Comment by nshanbhag20001412** [comment]
@Harshil-Jani  should i create a new branch and send the PR again?

**Comment by soumik2012** [comment]
@nshanbhag20001412  you can't  uncommit this please create a new PR

**Comment by Harshil-Jani** [comment]
There are two methods to deal with branch convolutions :
1. Easy Method : Go to your fork, Create new branch, Make required changes, Make a new Pull Request.
2. Advanced Method : 
If you know you want to uncommit changes, leaving everything else intact, use:
```git reset --soft HEAD^```
If you want to undo the act of committing and everything you'd staged, but leave the work tree (your files) intact:
```git reset HEAD^```
And if you actually want to completely undo it, throwing away all uncommitted changes, resetting everything to the previous commit 
```git reset --hard HEAD^```

Now, As this pull request is closed you can no more make commits on this. So, Only option left is 1st Method. But in future if you face this situation then you can make your choice.

Basically, @nshanbhag20001412 This Time, Create New branch, Add your Project and Create New Pull Request.

**Comment by nshanbhag20001412** [comment]
yes i did it. sent a PR
